documentation/modules/adding-source-provider.adoc

```diff
@@ -91,12 +91,19 @@ ifdef::vmware[]
 +
 
 . Choose one of the following options for validating CA certificates:
-
++
+** *Use a custom CA certificate*: Migrate after validating a custom CA certificate.
+** *Use the system CA certificate*: Migrate after validating the system CA certificate.
 ** *Skip certificate validation* : Migrate without validating a CA certificate.
-** *Use the system CA certificates*: Migrate after validating the system CA certificates.
-
-.. To skip certificate validation, select the *Skip certificate validation* check box.
-.. To validate the system CA certificates, leave the *Skip certificate validation* check box cleared.
++
+.. To use a custom CA certificate, leave the *Skip certificate validation* switch toggled to left, and _either_ drag the CA certificate to the text box _or_ browse for it and click *Select*.
+.. To use the system CA certificate, leave the *Skip certificate validation* switch toggled to the left, and leave the *CA certificate* text box empty.
+.. To skip certificate validation, toggle the *Skip certificate validation* switch to the right.
++
+. Optional, if you use a custom CA certificate or the system certificate: Fetch the CA certificate from the provider's API endpoint URL and confirm the validity of the certificate manually. Once confirmed, the CA certificate will be used to validate subsequent communication with the API endpoint.
```

Comment:
mentioning the `system certificate` at the beginning is confusing since one you fetch it this way, the CA certificate would be used as a custom CA certificate. I think this should start with something like "Optional, you can retrieve the customer CA certificate from the provider's API endpoint URL ..."
not sure we'll need the last part then, but if we keep it then it can change to something like: "Once confirmed, the CA certificate will be used as a custom CA certificate"

Reply:
@ahadas Does the new version solve the problem or do we still need to discuss custom CA vs. standard CA?  AFAIK, this section is the last open issue on the PR. 

What other providers does this section apply to?

Reply:
> Does the new version solve the problem or do we still need to discuss custom CA vs. standard CA? AFAIK, this section is the last open issue on the PR.

no, because it still says "if you use a custom CA certificate **or the system certificate**"
how about:
```suggestion
. Optional, you can ask the system to retrieve a custom CA certificate from the provider's API endpoint URL by pressing the  `Fetch certificate from URL` button. Then you will be asked to confirm the validity of the certificate manually. Once confirmed, the CA certificate will be used to validate subsequent communication with the API endpoint.
```
 
> What other providers does this section apply to?

it should apply also to RHV, OpenStack, and OpenShift (all but the OVA provider)